### DISCUSSION TITLE
Interpreting fitted effects related specifically to dispersion with GSR

### DISCUSSION THREAD

**Discussion by dc307** [bachlab/PsPM#731] [Q&A] [answered]
Hiya Dominik et al,

I am resurrecting a project from 2018, that used 4.0.1, and so have been reading through the tutorials and papers for PSPM, but I have a couple remaining questions I hope you can advise on.  We used DCM for GSR data that fit the amplitude, dispersion and latency to three sequential events per trial (these onsets are at least a couple seconds apart, but the flexible model boundaries has them right next to each other).  Anyway, we essentially have a 2 x 2 experiment.  One of the conditions is significantly predicted by the amplitude output and one is predicted by the dispersion and latency outputs.  All the papers and tutorials focus on amplitude, but I am wondering if the dispersion effect is meaningful on its own, or if it is just a parameter used to optimize the amplitude estimation - i.e. is the inference that we are able to predict the number of sudomotor bursts (dispersion of the gaussian) from our stimuli meaningful on its own? Are amplitude and dispersion independent? 

I suppose the second question is whether I need to be worried that we used 4.0.1 - are the outputs going to be the same in v6 - I am not particularly keen on reanalyzing for various practical reasons. 

Cheers
Dan

**Comment by dominikbach** [comment, marked as answer]
There is no publication yet on the meaning of the dispersion parameter, but you could extrapolate that from knowing that a Gaussian nerve pulse with 0.6 s dispersion, compared to one with 0.3 s dispersion, will cause an SCR with almost the same shape, but almosst twice the amplitude. (You can easily simulate that by creating such pulses, and using the matlab function `conv` to convolve them with the SCRF, which you can get by `scrf = pspm_bf_scrf_f(0.1)` where 0.1 is simply the time resolution which has to match the time resolution of your Gaussian pulse). Hence, to keep "classical" peak scoring comparable to DCM analysis with flexible dispersion, you'd have to multiply amplitude and dispersion parameters. The reason we haven't published on this so far is that it did not make a difference for our use case, fear conditioning - but in your case it may well do.

Regarding the difference between version 6.1.2 and 4.0.1 - the principle of DCM has not changed, and if you have no missing values in your data I suspect the results will be the same. What has changed is mainly the handling of missing values in DCM.

**Reply by dc307** [reply]
Hi Dominik,

In Staib, Castagnetti and Bach 2015 (Table 1) and Bach et al 2010 (Table 2), where you are predicting CS+ > CS-, and you are comparing the DCM approach to other analysis variations — what estimates from the inversion of the DCM did you use? Are you using just the amplitude estimates, or some combination of amplitude, dispersion, and latency to estimate sympathetic arousal (and if so, how are they combined?). Apologies if this is in the papers somewhere, or if it reflects a broader misunderstanding. I could not find it.

This may also seem like a silly question and it may be in the math but if the SA dispersion linearly predicts the amplitude of the SCR, assumedly similar to SA amplitude, how does pspm know what SCR variance to allocate to SA amplitude and what to allocate to SA dispersion? As you can guess, what I am getting at here, if there is an effect (as I have) associated with amplitude and a different effect associated dispersion (but not their combination), I need to know that there is a plausible reason for this when the real duration and amplitude of the SA is unknown.

If this is better suited to a chat, just let me know and I can email / chat direct. Thanks!

**Comment by dominikbach** [comment]
Hi Dan.

Previous publications just used the amplitude estimate as an index of conditioning. We haven't tried yet whether a combination of dispersion and amplitude might give a more accurate result (in the sense of, e.g. higher retrodictive validity, [10.3758/s13423-023-02421-z](https://doi.org/10.3758/s13423-023-02421-z).

Larger dispersion of the SN burst leads to larger amplitude - but also, longer duration of the ensuing SCR. The latter is how the non-linear model disentangles the two parameters. This works well in a noise-free setting but may fail when there is a lot of noise. We haven't done simulations on the separability of these two parameters under different noise regimes. In fact, we are now leaning towards fixing the dispersion, which is more plausible anyhow from a physiological perspective (using `options.constrained` in  `pspm_dcm`).

Dominik
